src/old/modules/materials/components/MaterialsView.tsx

```diff
@@ -158,15 +158,21 @@ const MaterialsView: React.FC = () => {
   };
 
   const fetchData = (appendPosts = false) => {
-    const originsQuery = query.get(QueryTypeEnum.ORIGINS)
-      ? query.get(QueryTypeEnum.ORIGINS)
-      : stringOfOrigins();
-    const postTypesQuery = query.get(QueryTypeEnum.POST_TYPES)
-      ? query.get(QueryTypeEnum.POST_TYPES)
-      : stringOfPostTypes();
-    const directionsQuery = query.get(QueryTypeEnum.DIRECTIONS)
-      ? query.get(QueryTypeEnum.DIRECTIONS)
-      : stringOfDirections();
+    const originsQuery =
+      query.get(QueryTypeEnum.ORIGINS) &&
```

Comment:
You can remove this check. If it's not present, you will get undefined and `query.get(QueryTypeEnum.ORIGINS) !== '0'` will handle it

Reply:
no, it doesn't do the trick :(